airflow/providers/amazon/aws/triggers/eks.py

```diff
@@ -33,49 +33,52 @@ class EksCreateFargateProfileTrigger(BaseTrigger):
 
     :param cluster_name: The name of the EKS cluster
     :param fargate_profile_name: The name of the fargate profile
-    :param poll_interval: The amount of time in seconds to wait between attempts.
-    :param max_attempts: The maximum number of attempts to be made.
+    :param waiter_delay: The amount of time in seconds to wait between attempts.
```

Comment:
Why all the renaming? The PR description doesn't mention this and it seems unrelated to factoring out the create compute method?

Reply:
I realized I've been naming these parameters incorrectly for the past few Triggers, and I was hoping to fix them. Considering the `EksCreateFargateProfileTrigger` got merged very recently, I was hoping to do this without doing a  deprecation warning. If it is problematic to include it as part of this PR, I can take it out.

Reply:
> got merged very recently, I was hoping to do this without doing a deprecation warning

This should be fine since we haven't done a provider release yet

 > If it is problematic to include it as part of this PR, I can take it out.
 
In a perfect world I think a separate PR would have been best, but it's not a huge deal, so let's keep it here for this time :+1: